apps/web/modules/bookings/views/bookings-single-view.tsx

```diff
@@ -163,6 +163,7 @@ export default function Success(props: PageProps) {
   const [calculatedDuration, setCalculatedDuration] = useState<number | undefined>(undefined);
   const [comment, setComment] = useState("");
   const parsedRating = rating ? parseInt(rating, 10) : 3;
+  const currentUserEmail = session?.user?.email ?? searchParams?.get("email");
```

Comment:
> We could have a fallback here in case neither of these values is available. Something like "Anonymous" or "Unknown User" might work

Will add

> We should also have this with URL params as the primary check, then session user, then "Anonymous".

I figured user can easily change the param and pretend to be someone else (can still do it if he does it in incognito), so enforced session user to be primary. Will change it.

Reply:
> We could have a fallback here in case neither of these values is available. Something like "Anonymous" or "Unknown User" might work

I just realised if we are defaulting to something like "Anonymous" or "Unknown User", won't it lead to validation issues in prisma and zod? 🤔